utils/presigner.go

```diff
@@ -180,5 +222,15 @@ func PresignGetCallerIdentity(validity time.Duration) (string, error) {
 
 	gu.RawQuery = nq.Encode()
 
+	if validity < DefaultValidity {
+		cacheValidity := validity
+		cacheValidity -= 1 * time.Minute
+		if cacheValidity < 0 {
+			cacheValidity = 1 * time.Minute
```

Comment:
why do we set it to 1 minute ? add comment maybe

I would expect to set it to `DefaultCacheTime` actually 🥲